src/Storage.Interface/Models/SignRequest.cs

```diff
@@ -15,6 +15,12 @@ public class SignRequest
         [JsonProperty(PropertyName = "signatureDocumentDataType")]
         public string SignatureDocumentDataType { get; set; }
 
+        /// <summary>
+        /// The task which should be linked to this signature
+        /// </summary>
+        [JsonProperty(PropertyName = "taskId")]
+        public string TaskId { get; set; } = string.Empty;
```

Comment:
@danielskovli Probably a bit late to mention, but you can read the current taskId from this property: `instance.Process.CurrentTask?.ElementId`

Reply:
Which begs the most important question: why is there no :facepalm: emoji available when reacting to comments?